tutorials/kubernetes/install-kubernetes-with-conjure-up.md

```diff
@@ -0,0 +1,159 @@
+
+### conjure-up
+
+conjure-up lets you summon up a big-software stack as a “spell” — a model of the stack, combined with extra know-how to get you from an installed stack to a fully usable one.
+
+### What you’ll learn
+  - How to install software using a conjure-up spell
+  - The simplest way to install The Canonical Distribution of Kubernetes
+
+### What you’ll need
+  - A Linux system with [snapd installed](https://snapcraft.io/docs/core/install)
+
+#### For cloud deployment
+  - Account credentials for one of the following public cloud providers:
+    - AWS
+    - Azure
+    - CloudSigma
+    - Google
+    - Joyent
+
+#### For local deployment
+
+Although this tutorial will assume a public cloud installation, if you wish to install locally the following minimum specifications are recommended:
+ - 2 cores
+ - 16G RAM
+ - 32G Swap
+ - 250G SSD with a separate block device for ZFS. Our recommendation for that device is 100G.
+
+## Getting started
+Duration: 5
+
+If you have not done so on your local machine, install conjure-up.
+
+On a Linux system with snapd simply type the following:
+
+```bash
+sudo snap install conjure-up --classic
+```
+
+positive
+: Note: If you need to install snapd on your Linux system, detailed instructions can be found in the [snapcraft.io documentation](https://snapcraft.io/docs/core/install).
+
+Once you have completed the conjure-up installation, run the software:
+
+```bash
+$ conjure-up
+```
+
+You will be presented with a list of recommended spells. From this list, select "The Canonical Distribution of Kubernetes" and press enter.
+
+![IMAGE](https://assets.ubuntu.com/v1/042c1dd7-select-canonical-distribution-of-kubernetes.png)
+
+## Select your cloud
+Duration: 3
+
+conjure-up will then ask where you would like to deploy your applications.
+
+For this tutorial we're going to demonstrate installation on a public cloud using Amazon AWS.
+
+There will be minor differences between the steps for the various cloud providers but generally consists of steps such as choosing a region and entering the credentials for the cloud provider.
+
+![IMAGE](https://assets.ubuntu.com/v1/43eb777a-Screenshot+from+2017-08-18+11-57-29.png)
+
+
+## Deploy the controller and applications
+Duration: 18
+
+Once you have configured the cloud provider that you are using you will then be presented with a screen titled "CHOOSE A CONTROLLER OR CREATE NEW".
+
+For the purposes of this tutorial we'll start from scratch and select "Deploy New Self-Hosted Controller".
+
+![IMAGE](https://assets.ubuntu.com/v1/ed7970a6-new-controller.png)
+
+You will be presented with a list of the applications required for a Kubernetes installation. You are presented with an option to individually configure the options but we can simply select "Deploy all remaining applictaions" and use the suggested default configuration.
```

Comment:
Suggestion to avoid repeating "option" in the same sentence and be more assertive. 

"You will be presented with a list of the applications required for a Kubernetes installation. Each of them can be configured individually, but for now, select "Deploy all remaining applications" to use the suggested default configuration."

Reply:
:+1: